packages/postman-to-openapi/rollup.config.ts

```diff
@@ -0,0 +1,12 @@
+import { createRollupConfig } from '@scalar/build-tooling'
+import type { RollupOptions } from 'rollup'
+
+const options: RollupOptions = {
+  input: ['./src/index.ts'],
+  ...createRollupConfig({
+    // Needed to enable explicit TypeScript (which Vite does not need)
+    typescript: true,
+  }),
+}
```

Comment:
No destructuring needed
```suggestion
const config = createRollupConfig({
  options: {
    input: ['src/index.ts'],
  },
  typescript: true,
})
```